.github/actions/javascript/checkDeployBlockers/checkDeployBlockers.ts

```diff
@@ -37,12 +38,12 @@ const run = function () {
 
             // If comments is undefined that means we found an unchecked QA item in the
             // issue description, so there's nothing more to do but return early.
-            if (_.isUndefined(comments)) {
+            if (!comments) {
```

Comment:
```suggestion
            if (comments === undefined) {
```

It's safer to do this way when dealing with `_.isUndefined`